### PR TITLE
implement command `DRYRUN`

### PR DESCRIPTION
**Pull request by ArtemIsmagilov**
- doc valkey command https://valkey.io/commands/acl-dryrun/
- doc redis command https://redis.io/docs/latest/commands/acl-dryrun/

### CODE REVIEW

File: redis/src/commands/mod.rs [hunk 1]
```diff
@@ -1109,6 +1109,13 @@ implement_commands! {
         cmd("ACL").arg("DELUSER").arg(usernames)
     }
 
+    /// Simulate the execution of a given command by a given user.
+    #[cfg(feature = "acl")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "acl")))]
+    fn acl_dryrun<P: ToRedisArgs>(username: P, command: P, args: &'a [P]) {
```
Comment (nihohit) [inline redis/src/commands/mod.rs:1115]: please give separate types for the arguments, to allow the user more freedom, and make sure they all implement ToRedisArgs

Comment (ArtemIsmagilov) [reply redis/src/commands/mod.rs:1115]: Thank you for commenting on my draft. I'll fix it now.

Comment (ArtemIsmagilov) [reply redis/src/commands/mod.rs:1115]: There may be one problem here though. Your Github Action checks for redis versions 7 and 6. However, this command is available from version 7. Well, let me correct the command first.

Comment (nihohit) [reply redis/src/commands/mod.rs:1115]: You can check the db's version and exit realty if it doesn't support the command. I'm not next to a computer right now, but look for is_major_version in tests

Comment (ArtemIsmagilov) [reply redis/src/commands/mod.rs:1115]: Thanks!

**Review by nihohit — APPROVED** [review]

**Pull request merged**